src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt

```diff
@@ -55,37 +51,43 @@ class ContextFilesPanel(
       val filePanel = createFileWithLinkPanel(contextFile)
       accordionSection.contentPanel.add(filePanel, index)
     }
+
+    this.removeAll()
+    this.isVisible = true
     add(accordionSection, BorderLayout.CENTER)
   }
 
   @RequiresEdt
   private fun createFileWithLinkPanel(contextFileFile: ContextFileFile): JPanel {
-    val anAction = OpenLocalContextFileAction(project, contextFileFile)
-    val actionText = contextFileFile.getLinkActionText(project.basePath)
-    val goToFile = AnActionLink(actionText, anAction)
-    goToFile.toolTipText = actionText
+    val anAction =
+        object : DumbAwareAction() {
+          override fun actionPerformed(anActionEvent: AnActionEvent) {
+            if (contextFileFile.repoName == null) {
```

Comment:
I see this check used multiple times, maybe we can add simple utility method for it like `isLocalFile`?

Reply:
done